src/game/World/WorldState.cpp

```diff
@@ -1860,7 +1864,7 @@ void WorldState::HandleActiveZone(uint32 attackTimeVar, uint32 zoneId, uint32 re
 
     // Calculate the next possible attack between ZONE_ATTACK_TIMER_MIN and ZONE_ATTACK_TIMER_MAX.
     uint32 zoneAttackTimer = urand(ZONE_ATTACK_TIMER_MIN, ZONE_ATTACK_TIMER_MAX);
-    TimePoint next_attack = now + std::chrono::milliseconds(zoneAttackTimer);
+    TimePoint next_attack = now + std::chrono::milliseconds(zoneAttackTimer * 1000);
```

Comment:
This should technically be
```cpp
TimePoint next_attack = now + std::chrono::seconds(zoneAttackTimer);
```
But I was getting some weird feedback in the "next attack in %d minutes" message, so I changed it back. Turns out that weird feedback is still present, so I guess this is superfluous